crates/codegen/parser/runtime/src/cst.rs

```diff
@@ -28,6 +28,7 @@ pub enum Node {
     Token(Rc<TokenNode>),
 }
 
+#[allow(clippy::match_wildcard_for_single_variants)]
```

Comment:
I wonder why allowing this here?

Reply:
Because the included impls most likely will match on the `cst::Node`, which is fairly stable and only has 2 variants; specifying the irrelevant variant in functions such as `as_*` is a noise. I can specify them inside, if you'd prefer.

Reply:
Done in 8582d28a8ba88b81e7907a255aef7e1c10408e58

Reply:
Thank you!